### PR TITLE
add agorapp to showcase.json and add agorapp svg logo

### PR DESCRIPTION
**Pull request by RiccardoBiosas**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

Notes:
The `video` field in showcase.json [points to our youtube video](https://github.com/dfinity/portal/compare/master...RiccardoBiosas:portal:master?expand=1#diff-106a38561050c1ad979ba999c303b207bd7af3fd6cd60aa3e691cc99400097d6R2438) because we weren't able to minify the grant submission's presentation enough.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @RiccardoBiosas,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/341) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -2419,5 +2419,26 @@
     "logo": "/img/showcase/motokopilot_logo.png",
     "screenshots": ["/img/showcase/motokopilot_screenshot.jpg"],
     "submittableId": "35960849"
+  },
+  {
+    "id": "agorapp",
+    "name": "AgorApp",
+    "oneLiner": "AgorApp is a Codecademy-syle platform that provides a browser-based IDE for all-things-web3-engineering. Learn how to build on the Internet Computer in an interactive environment.",
+    "website": "https://agorapp.dev/",
+    "tags": [
```
Comment (letmejustputthishere) [inline showcase.json:2428]: please only use tags available here
https://mvw4g-yiaaa-aaaam-abnva-cai.icp0.io/

**Review by letmejustputthishere — CHANGES_REQUESTED** [review]

File: showcase.json [hunk 2]
```diff
@@ -2527,6 +2527,26 @@
...
...e learned by solving their related coding exercises -no local setup needed. ICP builders can use AgorApp to learn how to build ICP DApps with Motoko, submit their code and then programmatically validate their solutions against the test-cases run by our infrastructure. Alongside courses, AgorApp provides CTF and algorithm challenges with bounty prizes: the users that hack the security challenges in the least amount of time or implement the most efficient implementation of an algorithm coding problem will be rewarded with ICP tokens and ICP native badges. AgorApp's fast feedback loop paired with our web3 workspace board which aggregates all grants/jobs/bounties/hackathons/events in the web3 ecosystem and recommends them to users based on their skillsets, github portfolio, AgorApp achievements and third-party integrations (e.g. Gitcoin, Dfinity, etc.) makes AgorApp the go-to place for developer onboarding. In the future, AgorApp will extend its educational-focused IDE to a low-code/rapid-prototyping end-to-end IDE with support for native ICP dev plugins (i.e.: fuzzers and static analyzers), a marketplace for web3 components and fine-tuned LLM for code completion and code generation.",
```
Comment (domwoe) [inline showcase.json:2539]: Could you please shorten the description to about 300-400 characters max?

Comment (RiccardoBiosas) [reply showcase.json]: Done

**Review by domwoe — APPROVED** [review]

**Pull request merged**